sample-btcd.conf

```diff
@@ -337,7 +350,7 @@
 ; You may also specify <subsystem>=<level>,<subsystem2>=<level>,... to set
 ; log level for individual subsystems.  Use btcd --debuglevel=show to list
 ; available subsystems.
-; debuglevel=info
+debuglevel=trace
```

Comment:
Keep the level as info. You can use trace locally but don't commit it.

Reply:
Fixed